end_to_end/signal_handler_test.go

```diff
@@ -30,7 +32,20 @@ var _ = Describe("Signal handler tests", func() {
 				"--single-data-file",
 				"--verbose"}
 			cmd := exec.Command(gpbackupPath, args...)
+			outPile, _ := cmd.StdoutPipe()
+			outScanner := bufio.NewScanner(outPile)
+			var stdout strings.Builder
+			var stderr strings.Builder
+			cmd.Stderr = &stderr
```

Comment:
how about
```suggestion
			stdout := cmd.createStdout()
			stderr := cmd.createStdErr()
```
where
```go
func (c *Cmd) createStdout() (io.Writer) {
	var b bytes.Buffer
	c.Stdout = &b
	return c.Stdout
}

func (c *Cmd) createStdErr() (io.Writer) {
	var b bytes.Buffer
	c.Stderr = &b
	return c.Stderr
}
```

Reply:
How do you propose to wait for output from stdout using Writer?

Reply:
rewrite by strings.Builder, main idea is reduce diff